searx/settings.yml

```
@@ -78,14 +78,18 @@ server:
   # public URL of the instance, to ensure correct inbound links. Is overwritten
   # by ${SEARXNG_URL}.
   base_url: false  # "http://example.com/location"
-  limiter: false  # rate limit the number of request on the instance, block some bots
-  public_instance: false  # enable features designed only for public instances
+  # rate limit the number of request on the instance, block some bots.
+  # Is overwritten by ${SEARXNG_LIMITER}
+  limiter: false
+  # enable features designed only for public instances.
+  # Is overwritten by ${SEARXNG_PUBLIC_INSTANCE}
+  public_instance: false
 
   # If your instance owns a /etc/searxng/settings.yml file, then set the following
   # values there.
 
   secret_key: "ultrasecretkey"  # Is overwritten by ${SEARXNG_SECRET}
-  # Proxying image results through searx
+  # Proxy image results through searx. Is overwritten by ${SEARXNG_IMAGE_PROXY}
```

```suggestion
  # Proxy image results through SearXNG. Is overwritten by ${SEARXNG_IMAGE_PROXY}
```
Not related to your changes, but since you're already on it :)

---

updated!